lib/reservation_monitor.py

```diff
@@ -101,24 +101,38 @@ def _check_flight_fares(self) -> None:
             try:
                 fare_checker.check_flight_price(flight)
                 self.notification_handler.healthchecks_success(
-                    f"Successful fare check, confirmation number={flight.confirmation_number}"
+                    f"Successful fare check,\nconfirmation number = {flight.confirmation_number}"
                 )
             except RequestError as err:
                 logger.error("Requesting error during fare check. %s. Skipping...", err)
                 self.notification_handler.healthchecks_fail(
-                    f"Failed fare check, confirmation number={flight.confirmation_number}"
+                    f"Failed fare check,\nconfirmation number = {flight.confirmation_number}"
                 )
             except FlightChangeError as err:
                 logger.debug("%s. Skipping fare check", err)
                 self.notification_handler.healthchecks_success(
-                    f"Successful fare check, confirmation number={flight.confirmation_number}"
+                    f"Successful fare check,\nconfirmation number = {flight.confirmation_number}"
                 )
             except Exception as err:
                 logger.exception("Unexpected error during fare check: %s", repr(err))
                 self.notification_handler.healthchecks_fail(
-                    f"Failed fare check, confirmation number={flight.confirmation_number}"
+                    f"Failed fare check,\nconfirmation number = {flight.confirmation_number}"
                 )
 
+    def _sleep_message(self, hours, minutes, seconds):
+        if self.first_name is not None and self.last_name is not None:
+            name = f" for {self.first_name} {self.last_name}"
+        else:
+            name = ""
+        
+        if hours > 0:
+            print(f"Sleeping for {hours} hours and {minutes} minutes until the next check{name}\n")
```

Comment:
I'm not a big fan of having this message printed. What's the reason for it? It's good for a debug message (which was already printed on line 130 before these changes). However, I don't think this isn't particularly useful to users. They already know it will check once per day and when this runs for enough time, the stdout will be filled with `Sleeping for 23 hours and 59 minutes until the next check for ...`.

Reply:
We can change this back to logger.debug but I wanted to have some visual when not using debug to show when the script is finished checking and is on standby. As an example, if we have 3 reservations in total (running in accounts or in reservation mode) after the last reseravtion to indicate script is not entering standby because if we don't run in debug it does shows checked in for the following flights and that's it. That's just my opinion, what do you think?

Reply:
> after the last reservation to indicate script is now entering standby because if we don't run in debug it only shows checked in for the following flights and that's it.

Could you expand on the situation here? I don't quite follow.

The script will show the messages for when all reservations are scheduled to check in, but I'm not sure if users would find it very helpful to print out a message every day on every account/reservation to indicate that it has just checked it (if desired, that can be received through healthchecks as well as the log). If you and/or others do find this helpful, though, I'm fine with keeping it as a print statement (but still logging the `Sleeping for ... seconds` debug message that was removed). 

If it's just going to be a logger.debug message, I think it's fine to leave it as the `Sleeping for ... seconds` instead of having multiple branches for a debug message. It's easy enough to parse while debugging and not super helpful to have hours and minutes printed out versus just seconds (only within the debug message).

Reply:
I agree with you not to print for every account and every reservation. I was thinking to print only once at the end - "Sleeping for xxx seconds" after all accounts and reservations have been completed.

Reply:
The only problem with that is that every account/reservation can sleep for a different period (retrieval_interval can be configured on a per-account and per-reservation basis) so there's not only one correct message. The best solution may be to print the one with the lowest number of seconds but there is no communication between processes so I don't see that being possible (or even just one of these messages, as you suggested, instead of one per process)

Reply:
When user is not using debug... We don't need to display the seconds. Just at the end when everything is done to display "Monitoring is complete until the next check" something like that. When not running in debug right now it does tell you if it's still processing or it's done.